app/models/reporting/matview.rb

```diff
@@ -0,0 +1,10 @@
+module Reporting
+  class Matview < ActiveRecord::Base
+    def self.refresh
+      ActiveRecord::Base.transaction do
+        ActiveRecord::Base.connection.execute("SET LOCAL TIME ZONE '#{Period::REPORTING_TIME_ZONE}'")
```

Comment:
Do we have a helper for this? If not its probably time to create one - `with_reporting_time`

Reply:
I'm actually hoping to refactor refreshes so that this is the _only_ place we'll need to use this. Can you think of another place we need to set the DB local time in the same way?

Reply:
We use this while [refreshing the other matviews](https://github.com/simpledotorg/simple-server/blob/master/app/services/refresh_materialized_views.rb#L37) right now. Once we merge the two reporting namespaces, we'll be able to keep the refresh in literally one place. 

This is a pretty weird/obscure operation used in a specific context, and we don't want to propagate the use of it by adding a helper for it.